### DISCUSSION TITLE
post/react-server-components/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#6] [Announcements]
# post/react-server-components/

공식 릴리즈 전인 리액트 서버 컴포넌트에 대해 알아보고 준비해 봅니다.

https://tech.kakaopay.com/post/react-server-components/

**Comment by jaeilnet** [comment]
감사합니다~!

**Comment by Kanghaseong** [comment]
좋은 글 감사합니다.

**Reply by gangpyono** [reply]
잘 읽었습니다. 좋은 글 감사합니다!

**Comment by choseonghwan91** [comment]
좋은글 감사합니다! 서버 컴포넌트에 대해 처음 듣고 이글을 추천받아서 읽었는데도 이해하기 편하게 작성된 글이네요!

**Comment by Programming-Seungwan** [comment]
아티클의 마지막 파트에서 리액트 서버 컴포넌트와 서버 사이드 렌더링의 차이에 대해서 짚어주신 부분이 핵심을 파악하는 데에 너무 큰 도움이 되었습니다. 감사합니다 :)

**Comment by Kidongg** [comment]
좋은 글 감사합니다 :)

**Comment by korkt-kim** [comment]
1

**Comment by shinwonse** [comment]
저도 RSC와 SSR의 차이점에 대해서 설명해주신 부분이 가장 인상 깊었습니다! 깊이 생각하지 않은 저를 반성하게 되네요
